content/sensu-enterprise-dashboard/2.9/configuration.md

```diff
@@ -404,5 +404,21 @@ to configure the dashboard for RBAC with LDAP.
 Please see the [RBAC for OIDC reference documentation][4] for information on how
 to configure the dashboard for RBAC with OpenID Connect (OIDC).
 
-[1]:  [1]:  ../dashboard/#what-is-a-sensu-datacenter/
-[2]: ../../../sensu-core/1.2/reference/configuration/#configuration-merging/
\ No newline at end of file
+[1]:  /sensu-enterprise-dashboard/2.11/dashboard/#what-is-a-sensu-datacenter
+[2]: /sensu-core/1.2/reference/configuration/#configuration-merging
+[3]:  /sensu-enterprise-dashboard/2.11/rbac/overview
+[4]:  /sensu-enterprise-dashboard/2.11/rbac/rbac-for-ldap
+[5]:  /sensu-enterprise-dashboard/2.11/rbac/rbac-for-github
+[6]:  /sensu-enterprise-dashboard/2.11/rbac/rbac-for-gitlab
+[7]:  /sensu-enterprise-dashboard/2.11/rbac/audit-logging
+[8]:  #dashboard-definition-specification
+[9]:  #configuration-scopes
+[10]: #configuration-merging
+[11]: #auth-attributes
+[12]: #audit-attributes
+[13]: #ldap-attributes
+[14]: #github-attributes
+[15]: #gitlab-attributes
+[16]: /sensu-enterprise-dashboard/2.11/rbac/overview/#rbac-for-the-sensu-enterprise-console-api
```

Comment:
Looks like these links are pointing to a different version of the same product's docs. Perhaps we should use relative link to stay within the same version?